### ISSUE TITLE
`skip_label` option not working with `check_box` and `radio_button`

### ISSUE THREAD

**Issue by protaskz-in** [shivam091/rails_bootstrap_form#37]
Hello,

I come across one issue that `skip_label` option isn't working in case of `check_box` and `radio_button` helpers.

This option, as per description should not show label on DOM at all.

Examples:

```erb
form.radio_button :gender, "male", bootstrap: {skip_label: true}
```

```html
<div class="mb-3">
  <div class="form-check">
    <input class="form-check-input" type="radio" value="male" name="user[gender]" id="user_gender_male" />
    <label class="form-check-label" for="user_gender_male">Gender</label>
  </div>
</div>
```

Thanks

**Comment by shivam091** [comment]
Thank you for reporting this issue.

**Comment by shivam091** [comment]
Thanks for reporting this issue. It will be fixed in upcoming `0.9.4` release.
